src/rustllvm/RustWrapper.cpp

```diff
@@ -165,8 +165,8 @@ extern "C" void LLVMRemoveFunctionAttrString(LLVMValueRef fn, unsigned index, co
 }
 
 // enable fpmath flag UnsafeAlgebra
-extern "C" void LLVMRustSetHasUnsafeAlgebra(LLVMValueRef Instr) {
-    unwrap<Instruction>(Instr)->setHasUnsafeAlgebra(true);
+extern "C" void LLVMRustSetHasUnsafeAlgebra(LLVMValueRef V) {
+    if(auto O = dyn_cast<Instruction>(unwrap<Value>(V))) O->setHasUnsafeAlgebra(true);
```

Comment:
Can you put the body on a second line, and call the variable `I` instead of `O`? (because it's not an operator, sadly)